### DISCUSSION TITLE
[A-L] (26 Gennaio 2024) Esercizio 6

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#404] [Esami A-L]
La seguente formula proposizionale in CNF è soddisfacibile? [5 punti]

Se si risponde "SI" definire un assegnamento che la soddisfa, se si risponde "NO" dimostrare l’insoddisfacibilità usando la regola di Risoluzione.

### Fila A

$$
    \Large \\{\ \\{\neg p,\ \neg q,\ r\\},\ \\{\neg p,\ q,\ s\\},\ \\{p,\ \neg q\\},\ \\{p,\ q,\ \neg r\\}\ \\}.
$$

### Fila B

$$
    \Large \\{\ \\{\neg q,\ r\\},\ \\{q,\ s\\},\ \\{\neg q, \neg r, \neg s\\},\ \\{\neg q, \neg r\\}\ \\}.
$$

**Comment by cek753** [comment]
A: Sat p=1, q=0, r=0, s=1
B: Sat q=0,s=1
